chef_master/source/supermarket_share_cookbook.rst

```diff
@@ -0,0 +1,90 @@
+=====================================================
+Share Cookbooks on the Chef Supermarket
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/supermarket_share_cookbook.rst>`__
+
+This guide will show you how to share a cookbook on the public `Chef Supermarket <https://supermarket.chef.io/>`__. The public Supermarket uses `Hosted Chef <https://manage.chef.io>`__, the Chef-as-a-service provider, for user authentication. You will need a Hosted Chef account to share cookbooks.
+
+.. note:: If you already use Hosted Chef as your Chef server, skip to the cookbook sharing method of your choice:
+
+          * `Stove </supermarket_share_cookbook.html#share-cookbooks-via-stove>`__
+          * `Knife </supermarket_share_cookbook.html#share-cookbooks-via-knife>`__
+
+Create a Hosted Chef Account
+-----------------------------------------------------
+
+#. Go to the `Hosted Chef signup page <https://manage.chef.io/signup>`__ and enter the requested information to create your account.
+
+#. You will receive a confirmation email. Use the link in the email to confirm your confirm your email address.
+
+#. Log in to Hosted Chef and click the **Create New Organization** button:
+
+   .. image:: ../../images/hosted_chef_welcome.png
+
+#. Download and extract the Hosted Chef starter kit:
+
+   .. image:: ../../images/download_starter_kit.png
+
+Share Cookbooks via Stove
+-------------------------------------------------------
+`Stove <https://github.com/sethvargo/stove>`__ is a cookbook release utility that keeps the upload process localized to the cookbook itself, as opposed to the `Knife </supermarket_share_cookbook.html#share-cookbooks-via-knife>`__ method which requires a cookbook repository.
+
+#. Add the following line to the ``Gemfile`` within your project's repository:
+
+   .. code-block:: ruby
+
+      gem 'stove'
```

Comment:
`stove` is included in the Chef DK. Is the intention of this guide to not require the user to download the whole of Chef DK?

Reply:
@robbkidd Nope. I actually didn't know stove was included in ChefDK - grabbed these instructions from the GH readme. 

I'll add the ChefDK method with a link to the readme for anyone who happens to not be using ChefDK.